src/app/public/modules/phone-field/types/country.ts

```diff
@@ -1,9 +1,32 @@
 export interface SkyPhoneFieldCountry {
+  /**
+   * Specifies the name of the country.
+   * @required
+   */
   name: string;
+  /**
+   * Specifies the country's [International Organization for Standardization Alpha 2]
```

Comment:
Sorry about this misplaced comment. Here's where the link isn't generating.

Reply:
Ah. In that case, I'm guessing its because of breaking the link up across 2 lines. I'll change that to see if fixes the link.